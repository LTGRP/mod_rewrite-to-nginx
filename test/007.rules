# From the nginx manual:
#
#   Note: for curly braces( { and } ), as they are used
#   both in regexes and for block control, to avoid
#   conflicts, regexes with curly braces are to be
#   enclosed with double quotes (or single quotes).
RewriteRule   ^(20[0-9]{2})-((0|1)[0-9])/?$     /date/$1/$2

