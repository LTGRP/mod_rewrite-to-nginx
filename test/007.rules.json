[
    {
        "type": "rewrite",
        "match": "^(20[0-9]{2})-((0|1)[0-9])/?$",
        "target": "/date/$1/$2"
    }
]
