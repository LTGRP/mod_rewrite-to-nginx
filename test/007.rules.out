rewrite "^(20[0-9]{2})-((0|1)[0-9])/?$" /date/$1/$2;
